packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -110,90 +109,55 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
   private processTranscription = (voiceaPayload: TranscriptionResponse): void => {
     switch (voiceaPayload.type) {
       case TRANSCRIPTION_TYPE.TRANSCRIPT_INTERIM_RESULTS:
-        Trigger.trigger(
-          this,
-          {
-            file: 'voicea',
-            function: 'processTranscription',
-          },
-          EVENT_TRIGGERS.NEW_CAPTION,
-          {
-            isFinal: false,
-            transcriptId: voiceaPayload.transcript_id,
-            transcripts: voiceaPayload.transcripts,
-          }
-        );
+        // @ts-ignore
+        this.trigger(EVENT_TRIGGERS.NEW_CAPTION, {
```

Comment:
Cant we add trigger in the VoiceaChannel class to avoid using @ts-ignore in the file .

Reply:
No. I don't think it can be done. We have to use AmpState if we have to as it extends from AmpState